chain-tx-enclave/enclave-ra/ra-client/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+//! This crate exposes types for connecting to an enclave over attested TLS connection.
```

Comment:
Yes. This can be moved there once we have attested TLS `tx-query` server that uses `ra-enclave`.

Reply:
this can be moved even before -- the TLS tx-query server on the "surface" would ideally behave the same way as the current one... so client can be moved and tested against the current one (assuming this https://github.com/crypto-com/chain/pull/1311#discussion_r398439429 is changed)

Reply:
No. This cannot be moved now. Because current `tx-query` server encodes things in different format.

Reply:
can this be made compatible? it'd make things easier and testable

Reply:
This cannot be made compatible with current version of `tx-query` because current version uses custom pipe (`|`) delimited encoding scheme ([here](https://github.com/crypto-com/chain/blob/master/chain-tx-enclave/tx-query/enclave/src/cert.rs#L55)) to encode attestation report in TLS certificate and with this change, we'll start using JSON.